tests/unit/modules/ddns_test.py

```diff
@@ -101,11 +101,11 @@ def mock_udp_query(*args, **kwargs):
 
         if six.PY3:
             # pylint: disable=E0598
-            with patch.object(dns.message, 'make_query', MagicMock(return_value=mock_request)), \
-                    patch.object(dns.query, 'udp', mock_udp_query()), \
-                    patch.object(dns.rdatatype, 'from_text', MagicMock(return_value=mock_rdtype)), \
-                    patch.object(ddns, '_get_keyring', return_value=None), \
-                    patch.object(ddns, '_config', return_value=None):
+            with contextlib.nested(patch.object(dns.message, 'make_query', MagicMock(return_value=mock_request)),
```

Comment:
`nested` is removed from `contextlib` in Python 3 (they use ExitStacks instead now) so this won't work, and was why I made this change originally. I didn't realize the `\` would cause issues in Python 2.6.